src/App.js

```diff
@@ -1,7 +1,8 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
 import {Navbar,Footer} from "./components/components";
-import {Login,Products,Signup} from "./pages/pages";
+import {Login,Products,Signup,Wishlist} from "./pages/pages";
```

Comment:
If you change the name of `pages.js` file then you can import from the pages folder without then need to mention the filename